EIPS/eip_195_pure_call.md

```diff
@@ -0,0 +1,23 @@
+### Parameters
+
+* BASE_GAS_COST: 80
+* CODE_BYTES_PER_GAS: 15
```

Comment:
Perhaps to add some more explanation: In EVM1.5, code that is about to be executed needs to be analyzed and sightly processed first. For regular contracts, this is done once at creation time and we can use the gas from the code deposit to pay for this preprocessing. Here, 15 bytes per gas could be too low for that.